package.json

```
@@ -149,7 +149,7 @@
     "rc-steps": "~4.1.0",
     "rc-switch": "~3.2.0",
     "rc-table": "~7.25.3",
-    "rc-tabs": "~11.16.0",
+    "rc-tabs": "^12.1.0-alpha.1",
```

```suggestion
    "rc-tabs": "~12.1.0-alpha.1",
```

feature branch please.

---

already changed and resolved